src/Crocodealer/ColorTerminal.hs

```diff
@@ -0,0 +1,37 @@
+module Crocodealer.ColorTerminal
```

Comment:
Created https://github.com/kowainik/crocodealer/issues/22 to use `colourista` instead :slightly_smiling_face: